riscv/processor.h

```diff
@@ -283,6 +296,8 @@ class processor_t : public abstract_device_t
   bool halt_on_reset;
   std::vector<bool> impl_table;
 
+  bool extension_disable_table[256] = {};
```

Comment:
To match recent changes to `isa_parser_t`, and to support more than 256 extensions, please use the data structure `std::bitset<NUM_ISA_EXTENSIONS>` instead of `bool[256]`.

Reply:
I think it would be easier to understand if we invert the polarity of this table (`extension_enable_table`).  This makes more sense when combined with my comment above https://github.com/riscv-software-src/riscv-isa-sim/pull/1176/files#r1092287066

Reply:
OK. I'll try to update these.